PixivArtistHandler.py

```diff
@@ -12,33 +12,28 @@
 import PixivHelper
 import PixivImageHandler
 from PixivException import PixivException
+from PixivListHandler import process_blacklist, process_list_with_db
 
 
 def process_member(caller,
                    config,
                    member_id,
-                   user_dir='',
                    page=1,
```

Comment:
Why `user_dir` is removed? It is used for option 4 (download from list), as it can define the custom path for each member.
![image](https://user-images.githubusercontent.com/836867/107134404-b0c35180-692c-11eb-9395-06e565152baf.png)